patches/chromium/revert_use_accessibility_pkey_when_setting_page_access.patch

```diff
@@ -12,151 +12,38 @@ This patch should be removed once
 https://chromium-review.googlesource.com/c/chromium/src/+/3949284
```

Comment:
Is this patch needed given the CL mentioned has been merged ?

Reply:
@deepak1556 it's not in this roll yet - given it's paused i'll assume we remove it in the next roll.

Reply:
@codebytere the change landed in `109.0.5410.0`, it should be available in this roll.

Reply:
@codebytere the CL has been merged and is in this roll.  This patch should now be removed.  Because this patch is a revert instead of a cherry pick the patch won't automatically disappear.  I added this patch as a revert because the upstream CL had dependent CLs.